.circleci/config/@config.yml

```diff
@@ -53,49 +53,49 @@ workflows:
           <<: *staging_branch_only_filter
       - system_tests:
           <<: *staging_branch_only_filter
-      - integration_tests:
-          requires:
-            - build_image
-          <<: *staging_branch_only_filter
-      - integration_tests_helm:
-          requires:
-            - build_image
-          <<: *staging_branch_only_filter
-      - integration_tests_proxy:
-          requires:
-            - build_image
-          <<: *staging_branch_only_filter
-      - eks_integration_tests:
-          requires:
-            - build_image
-          <<: *staging_branch_only_filter
-      - aks_integration_tests:
-          requires:
-            - build_image
-          <<: *staging_branch_only_filter
-      - integration_tests_operator_on_k8s:
-          requires:
-            - build_image
-            - build_and_upload_operator
-          <<: *staging_branch_only_filter
+      # - integration_tests:
+      #     requires:
+      #       - build_image
+      #     <<: *staging_branch_only_filter
+      # - integration_tests_helm:
+      #     requires:
+      #       - build_image
+      #     <<: *staging_branch_only_filter
+      # - integration_tests_proxy:
+      #     requires:
+      #       - build_image
+      #     <<: *staging_branch_only_filter
+      # - eks_integration_tests:
+      #     requires:
+      #       - build_image
+      #     <<: *staging_branch_only_filter
+      # - aks_integration_tests:
+      #     requires:
+      #       - build_image
+      #     <<: *staging_branch_only_filter
+      # - integration_tests_operator_on_k8s:
```

Comment:
I <think> this can be fully removed. @kat1906 can you confirm? :-/

Reply:
Let's disable it now now, and have the removing part in a different PR 🙏🏻

Reply:
I haven't started working on that et so can stay for now :)